src/parquet/column/properties.h

```diff
@@ -81,8 +82,11 @@ static int64_t DEFAULT_PAGE_SIZE = 1024 * 1024;
 static int64_t DEFAULT_DICTIONARY_PAGE_SIZE = DEFAULT_PAGE_SIZE;
 static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
 static Encoding::type DEFAULT_ENCODING = Encoding::PLAIN;
+static Encoding::type DEFAULT_DICTIONARY_ENCODING = Encoding::PLAIN_DICTIONARY;
```

Comment:
In V1, both Dictionary and Data Pages have `PLAIN_DICTIONARY`encoding, but the dictionary values are always plain encoded and indices are RLE encoded.
In V2, they made it more clearer by setting Dictionary Pages to `PLAIN` encoding and Data Pages to `RLE_DICTIONARY`.